nixos/modules/services/hardware/tlp.nix

```diff
@@ -37,6 +37,21 @@ in
 
   ###### implementation
   config = mkIf cfg.enable {
+    warnings = with builtins.isNull;
+    if !isNull config.powerManagement.scsiLinkPolicy then
+      [ "You've set a non-null value for powerManagement.scsiLinkPolicy. The TLP module will override that decision." ]
```

Comment:
This warning seems inaccurate to me.
Either the user didn't use mkForce, and the warning does not fire because the config is overriden by the module.
Or the user used something stronger than mkForce, and the "The TLP module will override that decision" part of the sentence is false.

The same applies below.